libs/partners/unstructured/langchain_unstructured/document_loaders.py

```diff
@@ -88,14 +88,14 @@ def __init__(
         # SDK parameters
         api_key: Optional[str] = None,
         client: Optional[UnstructuredClient] = None,
-        server_url: Optional[str] = None,
+        url: Optional[str] = None,
```

Comment:
breaking change

Reply:
I'm ok with this as a 0.1.1 release, and a yank of 0.1.0 - this is currently an unannounced package.

Reply:
I dropped the commit that changed it back to `server_url` and updated the version to 0.1.1, so the PR should be good to go now @efriis